integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -151,50 +160,32 @@ public void testCreateSecondOperatorUsingSameOperatorNsNegativeInstall() throws
     Operator secondoperator = new Operator(operatorMap, false, true, true, RestCertType.SELF_SIGNED);
     String oprelease = (String)(secondoperator.getOperatorMap()).get("releaseName");
     String opnamespace = (String)(secondoperator.getOperatorMap()).get("namespace");
+    boolean installFailed = true;
+    boolean gotExpectedInfo = true;
     try {
       secondoperator.callHelmInstall();
-      throw new RuntimeException(
-          "FAILURE: Helm installs second operator with same namespace as the first one");
-
+      installFailed = false;
     } catch (Exception ex) {
       if (!ex.getMessage()
-          .contains(
-              "Error: release "
-                  + oprelease
-                  + " failed: secrets \"weblogic-operator-secrets\" already exists")) {
-        throw new RuntimeException(
-            "FAILURE: Helm installs second operator with same namespace as the first one "
-                + "does not report expected message "
-                + ex.getMessage());
-      }
-      String cmdLb = "";
-      if (BaseTest.HELM_VERSION.equals("V2")) {
-        cmdLb = "helm list --failed " + "  | grep " + oprelease;
-      }
-      if (BaseTest.HELM_VERSION.equals("V3")) {
-        cmdLb = "helm list --failed --namespace " + opnamespace + "  | grep " + oprelease;
-      }
-      LoggerHelper.getLocal().log(Level.INFO, "Executing cmd " + cmdLb);
-      ExecResult result = ExecCommand.exec(cmdLb);
-      if (result.exitValue() != 0) {
-        throw new RuntimeException(
-            "FAILURE: Helm installs second operator with same namespace as the first one ");
+          .contains("weblogic-operator-secrets")) {
```

Comment:
I don't understand this and it seems brittle.  If the Helm install is supposed to fail, I would expect a non-zero exit code from invoking the `helm` binary and that the list of installed Helm charts does not include the chart we just tried to install.

Reply:
Here the callHelmInstall() expects to fail with a proper exception with the following message

Error: rendered manifests contain a resource that already exists. Unable to continue with install: existing resource conflict: kind: Secret, namespace: ons, name: weblogic-operator-secrets